src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts

```diff
@@ -32,7 +40,23 @@ export class GbVariablesComponent implements OnInit {
         this.categorizedVariables.get(variable.type).push(variable);
       }
     });
-    console.log('categorized variables: ', this.categorizedVariables);
+  }
+
+  onCheckAll(e) {
+    this.checkAllText = e ? 'All selected' : 'None selected';
```

Comment:
Having "All selected" or "None selected" test above the variables is a bit confusing, when you select/deselect something not using `CheckAll` checkbox, but a checkbox next to a specific variable. Maybe better to have a counter there as well as a third option: "`x` selected" or at least hide the text at all, when not all/none is selected.

Reply:
ah, right, never thought of it. will change it!